packages/features/bookings/lib/booking-report.repository.ts

```diff
@@ -0,0 +1,59 @@
+import type { PrismaClient } from "@calcom/prisma";
```

Comment:
I think this repository should live with others in server/repository as PrismaBookingReportRepository